### PR TITLE
Sage images for CO, SQ, and PS

### PR DESCRIPTION
**Pull request by cg2wilson**

### CODE REVIEW

**Review by cg2wilson — COMMENTED** [review]
i'm having trouble getting the `polarCoords` image to compile in CO4. it compiles fine in cocalc, but it won't compile in codespaces, and it was identified as a problem from a terminal build.

**Comment by siwelwerd** [comment]
Was getting some odd build errors I think related to the new pretext version that was fixed in the meantime, so tried to update this with main to incorporate those fixes to see if that helps

**Comment by siwelwerd** [comment]
> it compiles fine in cocalc, but it won't compile in codespaces, and it was identified as a problem from a terminal build.

Maybe I should have tried this first, but I immediately got an error copying + pasting your sage code into a sagecell.  So I always recommend t his as a trouble shooting step.  It complains that you are using a function "lazy_string" that it doesn't know about (presumably it is from some library).  I think this is just duplicating what can be done with an f string, waiting on that to compile in PreTeXt to see if that fixes it.

**Comment by siwelwerd** [comment]
How's this look?

image

File: source/calculus/source/07-CO/04.ptx [hunk 1]
```diff
@@ -47,13 +63,24 @@
...
+plt = Graphics()
+for i in circ_range:
+    plt += polar_plot(i-0.5,x,0,2*pi,thickness=1,figsize=10)
+    plt += polar_plot(i,(x,0,2*pi), axes=False, thickness=1.5,figsize=10)
+    bbox = {'boxstyle': 'square,pad=0.05','fc':"white", 'ec':"white"}
+    lbl = text(i,(i,-.25),color='black',background_color='white',bounding_box = bbox,fontsize=13)
+    plt += lbl
+for angle in [0,1,..,23]:
+    plt += line(([(0,0),(5*cos(angle*pi/12),5*sin(angle*pi/12))]))
+    plt += text(f"${latex(angle*pi/12)}$",(5.5*cos(angle*pi/12),5.5*sin(angle*pi/12)),color='black',fontsize=13)
```
Comment (siwelwerd) [inline source/calculus/source/07-CO/04.ptx:77]:
```suggestion
    actual_angle = angle*pi/12
    if denominator(actual_angle) != 1:
        plt += text(f"$\\dfrac{{{latex(numerator(actual_angle))}}}{{{denominator(actual_angle)}}}$",(5.5*cos(angle*pi/12),5.5*sin(angle*pi/12)),color='black',fontsize=13)
    else:
        plt += text(f"${latex(angle*pi/12)}$",(5.5*cos(angle*pi/12),5.5*sin(angle*pi/12)),color='black',fontsize=13)
```

Better formatted labels:
image

**Comment by cg2wilson** [comment]
> How's this look?

Amazing is how it looks. Thanks for cleaning it up and fixing the compile issue!

**Comment by siwelwerd** [comment]
If you want to go ahead and merge this in, mark it as ready for review. Otherwise you can keep on working on this branch.

**Review by siwelwerd — APPROVED** [review]

**Pull request merged**